dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/ds3panel/ds3treetable/Ds3TreeTableValue.java

```diff
@@ -147,7 +147,7 @@ public String getDirectoryName() {
             case Bucket:
                 return StringConstants.EMPTY_STRING;
             default:
-                return getParentDir(this.getFullName());
+                return getParentDir(this.getFullName()) + "/";
```

Comment:
Is there any instance where getFullName will return a name with an non-encoded / already at the end?

Reply:
Not really, do you think that's a better place for it?